src/openrct2/rct2/S6Importer.cpp

```diff
@@ -141,6 +141,11 @@ class S6Importer final : public IParkImporter
             }
         }
 
+        if (_s6.header.classic_flag == 0xf)
```

Comment:
Hmm, do we know whether some parks have the extra data but do not have this set to 0x0F?

Maybe this would be safer:
`if (_s6.header.classic_flag & (1 << 3))`

Reply:
Hm, I would argue that would imply it actually works like bit flag. And I'm not sure if it does, since on a compatible save it is 0x7. I have tested it with two separate files that went over the limit and both have this property set to 0xf.

With it not being a bit flag (or at least, that's what I think), I'd prefer `> 7`. But perhaps we should examine more RCTC saves and see if there are versions out there having flags other than 0x7 or 0xf.